components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilder.kt

```diff
@@ -35,7 +35,7 @@ class WebExtensionBrowserMenuBuilder(
     private val store: BrowserStore,
     @ColorRes private val webExtIconTintColorResource: Int = NO_ID,
     private val onAddonsManagerTapped: () -> Unit = {},
-    private val appendExtensionSubMenuAtStart: Boolean = false
+    private val usingBottomToolbar: Boolean = false
```

Comment:
I'm not sure about the name change, as it's very specific to an app functionality, the menu should be abstracted from outside functionality. `appendExtensionSubMenuAtStart` couldn't tell us if we should put the submenu at the end or at the beginning of the list as the docs states?  
```
appendExtensionSubMenuAtStart true if web extension sub menu appear at the top (start) of
 the menu, false if web extensions appear at the bottom of the menu. Default to false (bottom).
```

Reply:
The reason for the change is that we no longer append the menu at the top or at the bottom. I just kept that option as a fallback if we do not have a placeholder for the menu. So, if we have a placeholder , that boolean would no longer have sense.

Reply:
Could we use a more generic name ?

Reply:
I could keep the old naming and modify the documentation to state that the property for placement at the top or bottom it's followed only when we do not have a placeholder (because adding one is a more explicit action). And also state that this `appendExtensionSubMenuAtStart ` value is also used to place the back button inside the extensions list. What do you think?

Reply:
Sounds good 👍🏽